app/views/jobs/_form.html.erb

```diff
@@ -8,6 +8,8 @@
   <%= f.label :level_of_interest %>
   <%= f.number_field :level_of_interest %>
 
+  <%= f.select(:category,  Category.all.map{ |f| [f.title, f.id ]}, {prompt: 'Pick a Category'}, id: "category") %>
```

Comment:
does this query (`Category.all`) belong in the view? what part of MVC is responsible for grabbing our data?